src/compliance-rules.js

```diff
@@ -2,7 +2,7 @@ import tippy from "tippy.js";
 import "tippy.js/dist/tippy.css";
 
 const registeredBpmnElements = new Map();
-const bpmnContainerElt = window.document.getElementById("bpmn-container");
+const bpmnContainerElt = window.document.getElementById("main-bpmn-container");

```

Comment:
⚠️ the HTML element should be retrieved directly from the BpmnVisualization instance. #13 broke the current code as the id of the bpmn-container changed.
This can be done with 
```typescript
const bpmnContainerElt = bpmnVisualization.graph.container;
```
This means that the logic should be change as well as the instance is not available globally.

Reply:
I am going to manage it in a dedicated Pull Request